### PR TITLE
Update docs about listing inputs

### PR DESCRIPTION
**Pull request by olinoskar**
I was considering whether to remove the old docs or keep it and mark it as deprecated. I decided to go for removing it since it will since I think it will incentivise users to migrate. One thing that's apparent is that we don't have any method to get inputs by uuids which would make a lot of sense to have

### CODE REVIEW

File: docs-src/docs/kognic-io/working_with_scenes_and_inputs.md [hunk 1]
```diff
@@ -138,49 +138,38 @@ The response is a list of `Scene` objects containing the following properties
...
 ### Response
 The response is a list of `Input` objects containing the following properties
 
 | Property         | Description                                                                                          |
 |------------------|------------------------------------------------------------------------------------------------------|
-| scene_uuid       | ID used to identify the scene within the Kognic Platform                                             |
-| external_id      | External ID supplied during scene creation                                                           |
-| batch            | Which batch does the input belong to                                                                 |
+| uuid             | ID used to identify the input within the Kognic Platform                                             |
+| scene_uuid       | ID used to identify the scene that the input using                                                   |
```
Comment (Jaxing) [inline docs-src/docs/kognic-io/working_with_scenes_and_inputs.md:168]:
```suggestion
| scene_uuid       | ID used to identify the scene that the input is using                                                   |
```

**Comment by Jaxing** [comment]
I think that not writing out deprecated docs will help not getting more users for the deprecated features, however I think that if it's clear that it's deprecated and support will vanish soon it will be more motivating that if the docs suddenly disappear which they might not even notice.

Also not for this pr but you can [version your docs](https://docusaurus.io/docs/versioning) with docusaurus which might be nice

**Comment by olinoskar** [comment]
> Also not for this pr but you can [version your docs](https://docusaurus.io/docs/versioning) with docusaurus which might be nice

That would be nice to try during a hack day or similar

**Review by petern-annotell — APPROVED** [review]
Looks nice!

**Pull request merged**